autoblocks/_impl/api/client.py

```diff
@@ -408,3 +408,28 @@ def get_human_review_job_pair(self, job_id: str, pair_id: str) -> AutoblocksTest
             chosen_output_id=pair.get("chosenOutputId", None),
             test_case_results=test_case_results,
         )
+
+    def add_dataset_item(self, dataset_external_id: str, data: Dict[str, Any]) -> str:
+        response = self._client.post(
+            f"/datasets/{encode_uri_component(dataset_external_id)}/items",
+            json={"data": data},
+        )
+        response.raise_for_status()
+        return str(response.json()["id"])
+
+    def delete_dataset_item(self, dataset_external_id: str, item_id: str) -> str:
+        response = self._client.delete(
+            f"/datasets/{encode_uri_component(dataset_external_id)}/items/{encode_uri_component(item_id)}"
+        )
+        response.raise_for_status()
+        return str(response.json()["id"])
+
+    def update_dataset_item(
+        self, dataset_external_id: str, item_id: str, data: Dict[str, Any], split_names: List[str]
+    ) -> str:
```

Comment:
let's make sure create and update have the same params, i think we just need to add split names to add dataset item.